types/hapi/index.d.ts

```diff
@@ -1619,7 +1632,7 @@ export interface ServerConnection {
     inject(options: string | InjectedRequestOptions, callback: (res: InjectedResponseObject) => void): void;
     inject(options: string | InjectedRequestOptions, ): Promise<InjectedResponseObject>;
     /** Described in server.table [See docs](https://hapijs.com/api/16.1.1#servertablehost) */
```

Comment:
The assumption I made is wrong and confusing.  Could you change this to the following please:

`/** Mentioned but not documented under server.connections [See docs](https://hapijs.com/api/16.1.1#serverconnections) */`